imap_processing/cdf/config/imap_glows_l1b_variable_attrs.yaml

```diff
@@ -401,14 +401,14 @@ spacecraft_velocity_average:
   FORMAT: I4
   LABLAXIS: Avg spacecraft velocity
 
-spacecraft_velocity_variance:
+spacecraft_velocity_std_dev:
   <<: *support_data_defaults
   DEPEND_1: ecliptic
-  CATDESC: spin-block averaged variance of spacecraft velocity
-  FIELDNAM: Variance of spacecraft velocity
+  CATDESC: spin-block averaged std_dev of spacecraft velocity
+  FIELDNAM: std_dev of spacecraft velocity
   FILLVAL: *max_uint16
   FORMAT: I4
-  LABLAXIS: Variance
+  LABLAXIS: std_dev
```

Comment:
is it just me or they change these names often? I feel like I have seen this before :)

Reply:
My fault, they changed the names once but I got confused and changed them back :) this is me actually changing them for real this time 😆